package.json

```diff
@@ -114,44 +114,43 @@
     "@types/js-yaml": "3.5.29",
     "@types/lodash": "4.14.58",
     "@types/mime": "0.0.29",
-    "@types/node": "7.0.12",
+    "@types/node": "^9.6.2",
```

Comment:
Should you change the package.json node dependency as well?  Right now it says
`"node": ">=6.9.4`